Contributing.md

```diff
@@ -21,6 +21,8 @@ When submitting a pull request, please follow these guidelines:
 
 - there should be no errors when running `npm run scan` or `yarn scan`
 - ensure your dependencies have been installed and that any/all file(s) changed have had prettier-apex run on them (usually as simple as enabling "Format On Save" in your editor's options); alternatively you can always format the document in Vs Code using `Shift + Ctrl + F` or `cmd + Shift + F` once you're done writing. Your mileage may vary as to the hotkey if you're using Illuminated Cloud or another text editor; you always have the option of invoking prettier on the command-line
-- ensure that tests have been run against a scratch org.
+- ensure that tests have been run against a scratch org with multi-currency enabled (you can look at `scripts/test.ps1` to see how the scratch org is created and the currency ISO codes are loaded).
```

Comment:
Thoughts on making this a link to the script?

```suggestion
- ensure that tests have been run against a scratch org with multi-currency enabled (you can look at [`scripts/test.ps1`](https://github.com/jamessimone/apex-rollup/blob/main/scripts/test.ps1) to see how the scratch org is created and the currency ISO codes are loaded).
```

Reply:
Brilliant - I love it